spec/lib/danger/danger_core/dangerfile_spec.rb

```diff
@@ -158,7 +158,7 @@ class DangerCustomAttributePlugin < Danger::Plugin
 
       dm = testing_dangerfile
       methods = dm.external_dsl_attributes.map { |hash| hash[:methods] }.flatten.sort
-      expect(methods).to eq [:my_thing, :protect_files]
+      expect(methods).to eq [:my_thing]
```

Comment:
`:protect_files` leaks in from our `danger_plugins` dir, glad to be rid of it now 👍